ChustaSoft.Tools.Authorization/Configuration/AuthorizationSettingsBuilder.cs

```diff
@@ -56,11 +60,16 @@ public AuthorizationSettingsBuilder SetMinutesToExpire(int minutesToExpire)
             return this;
         }
 
-        public AuthorizationSettings Build() => _authorizationSettings;
+        public AuthorizationSettingsBuilder AddExternalAuthorization(ExternalAuthenticationProviders providerName, ExternalAuthenticationSettings externalAuthenticationSettings)
```

Comment:
Suggestions here:

This is perfectly valid, but we can add another configuration option, Keep this as it is, and add a new AddExternalAuthorization overloaded method with the following parameters:

- providerName (ExternalAuthenticationProviders)
- clientId (string)
- secretId (string)

This alllow the user not only to send a new ExternalAuthenticationSettings, also can ease configuration sending parameters separated

Reply:
Overloaded method added